app-config.yaml

```diff
@@ -4,6 +4,11 @@ app:
 
 backend:
   baseUrl: http://localhost:7000
+  bindPort: 7000
+  bindHost: localhost
+  cors:
+    origin: http://localhost:3000
+    credentials: true
```

Comment:
this file contains some duplication at this point, because each part is locally specialised for its needs - for example, we could conceivably extract `backend.bindPort` from `backend.baseUrl`, and set `backend.cors.origin` to `app.baseUrl` or whatever - but they will actually vary under some circumstances so they will stay separate for now

Reply:
Was also thinking that grabbing cors origin from `app.baseUrl` would be good, can't see any reason it would be different?

Reply:
I'm just a little concerned that we would be tying unrelated parts of the config to each other. Makes it harder to evolve.

Reply:
yeah got no problem with keeping it explicit for now, but I'm also dreading the 1000-line config ppl have to write to spin up an instance.